docs/AboutUs.md

```diff
@@ -2,7 +2,7 @@
 
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | Liew Wei Siew | [Github](https://github.com/LiewWS) | [Portfolio](./team/LiewWS.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Liew Wei Siew | [Github](https://github.com/LiewWS) | [Portfolio](./team/liewws.md)
```

Comment:
I think you can add your picture here and link it.

Reply:
Alright thanks for the suggestion.